amps/ags/rm-community/rm-community-repo/scripts/prepare_buildx.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+LOCAL_REGISTRY_HOST="${1}"
+LOCAL_REGISTRY_PORT="${2}"
+BASE_IMAGE="${3}"
+BASE_IMAGE_TAG="${4}"
+BUILDER_NAME="${5}"
+LOCAL_REGISTRY="${LOCAL_REGISTRY_HOST}":"${LOCAL_REGISTRY_PORT}"
+
+#Run a local registry server
+docker run -d -p "${LOCAL_REGISTRY_PORT}":"${LOCAL_REGISTRY_PORT}" --restart=always --name registry registry:2
+
+#Push base image to the local repository
+docker tag "${BASE_IMAGE}":"${BASE_IMAGE_TAG}" "${LOCAL_REGISTRY}"/"${BASE_IMAGE}":"${BASE_IMAGE_TAG}"
+while [ "$( docker container inspect -f '{{.State.Running}}' registry )" != "true" ]
```

Comment:
We should add a reasonable timeout condition here to fail the script in case we wait on the registry to come up for too long.

Reply:
I added a timeout condition (2 minutes).